packages/babel-plugin-proposal-decorators/src/transformer-2022-03.ts

```diff
@@ -115,26 +119,20 @@ function replaceClassWithVar(
 
     return [t.cloneNode(varId), path];
   } else {
-    let className: string;
     let varId: t.Identifier;
 
     if (path.node.id) {
       className = path.node.id.name;
       varId = path.scope.parent.generateDeclaredUidIdentifier(className);
       path.scope.rename(className, varId.name);
-    } else if (
-      path.parentPath.node.type === "VariableDeclarator" &&
-      path.parentPath.node.id.type === "Identifier"
-    ) {
-      className = path.parentPath.node.id.name;
```

Comment:
Here is how we previously handle the namedEvaluation in the case `var c = @dec class {}`.